library/src/main/java/com/google/android/exoplayer/audio/AudioTrack.java

```diff
@@ -1280,8 +1280,8 @@ public AudioTrackUtil(boolean isLatencyQuirkEnabled,
       this.getLatencyMethod = getLatencyMethod;
       try {
         getTimestamp = AudioTrack.class.getMethod("getTimestamp", AudioTimestamp.class);
-      } catch (Exception e) {
-        // There's no guarantee this method exists. Do nothing.
+      } catch (Throwable e) {
```

Comment:
Thomas,
Is the exception not already sufficient to handle the missing getTimeStamp?
if the getTimeStamp method doesn't exist, there will be a "NoSuchMethodException" thrown (it is being accessed via reflection). This exception is already handled in the catch block.